arangod/Cluster/FollowerInfo.h

```diff
@@ -273,8 +273,9 @@ class FollowerInfo {
           << "Shard " << _docColl->name()
           << " is temporarily in read-only mode, since we could not update the "
              "failover candidates in the agency.";
+      return WriteState::FORBIDDEN;
```

Comment:
Wasn't the idea to return `WriteState::UNAVAILABLE` here?

Reply:
done, thx